framework/src/modules/token/method.ts

```diff
@@ -708,4 +708,15 @@ export class TokenMethod extends BaseMethod {
 			})),
 		};
 	}
+
+	public async escrowSubstoreExists(
+		methodContext: MethodContext,
+		chainID: Buffer,
+		tokenID: Buffer,
+	) {
+		const escrowStore = this.stores.get(EscrowStore);
+		const escrowExist = await escrowStore.has(methodContext, escrowStore.getKey(chainID, tokenID));
+
+		return escrowExist;
```

Comment:
```suggestion
		return this.stores.get(EscrowStore).has(methodContext, escrowStore.getKey(chainID, tokenID));
```
probably better to be succinct similar to `isTokenSupported` method in the same file